tests/test_parser.py

```diff
@@ -1212,10 +1212,10 @@ def test_define_dict_attribute():
 
 
 def test_lexer():
+    # // comments are no longer supported
```

Comment:
This message refers to a change in this PR, not to anything related to the resulting test. I think it should go, because it's meaning is unclear.